### PR TITLE
Added has method on Model

### PR DESCRIPTION
**Pull request by maantje**
Allows to check if a record exists for a relation

### CODE REVIEW

File: src/Illuminate/Database/Eloquent/Model.php [hunk 1]
```diff
@@ -1255,6 +1255,23 @@ public function setPerPage($perPage)
     }
 
     /**
+     * Check if the model has atleast one record of a relation.
```
Comment (browner12) [inline src/Illuminate/Database/Eloquent/Model.php]: space between 'atleast'

Comment (maantje) [inline src/Illuminate/Database/Eloquent/Model.php]: Thanks, ill fix it now!

**Comment by joshmanders** [comment]
No tests?

**Comment by taylorotwell** [comment]
No plans to add this.

**Pull request closed without merging**